container.go

```diff
@@ -824,9 +824,9 @@ func (c *Container) removeDrive() (err error) {
 	return nil
 }
 
-func (c *Container) attachDevices() error {
+func (c *Container) attachDevices(create bool) error {
 	for _, device := range c.devices {
-		if err := device.attach(c.pod.hypervisor); err != nil {
+		if err := device.attach(c.pod.hypervisor, c, create); err != nil {
```

Comment:
Here as well, you may only need to pass the container pointer and infer from its state if it's already running (thus hotplug) or not.